talisker/__init__.py

```diff
@@ -87,6 +87,23 @@ def get_config(env=os.environ):
     }
 
 
+TALISKER_ENV_VARS = {
+    # development
+    'DEVEL',
+    'DEBUGLOG',
+    'TALISKER_COLOR',
+    'TALISKER_LOGSTATUS'
+    # sentry config
+    'SENTRY_DSN',
```

Comment:
On the first service I looked at (snapauth), `SENTRY_DSN` contains a username and password.  I can see how it's useful to log part of this, but you should strip authentication information from the URL.

Reply:
Aha, right you are. I all ready have code to do that for logging sentry config, so I can reuse that. Good catch.

Reply:
Hmm, do you think we should strip username and password, or just password?

Reply:
Just password would be fine.